README.md

```diff
@@ -225,8 +225,8 @@ end
 Route patterns may have optional parameters:
 
 ``` ruby
-get '/posts.?:format?' do
-  # matches "GET /posts" and any extension "GET /posts.json", "GET /posts.xml" etc.
+get '/post/:format?' do
+  # matches "GET /posts" and any extension "GET /posts/json", "GET /posts/xml" etc
```

Comment:
I believe most HTTP servers automatically mismatch the trailing ‘/‘, which is how the /folder/index.html trick works as a /folder url (e.g. with middleman), or at least thats what I was lead to believe when I picked the example :)  

Jon Rowe
---------------------------
mail@jonrowe.co.uk
jonrowe.co.uk


On Saturday, 15 August 2015 at 17:35, Katrina Owen wrote:

> In README.md (https://github.com/sinatra/sinatra/pull/1025#discussion_r37137119):
> > @@ -225,8 +225,8 @@ end > Route patterns may have optional parameters: > > ``` ruby > -get '/posts.?:format?' do > - # matches "GET /posts" and any extension "GET /posts.json", "GET /posts.xml" etc. > +get '/post/:format?' do > + # matches "GET /posts" and any extension "GET /posts/json", "GET /posts/xml" etc  
> Ah, right. Damn, I missed that.
> OK, so if we do get '/posts/:year?, then the comment would have to say that it `matches "GET /posts/", which sucks as a route, since people typing things in would tend to leave off the trailing slash.
>  
> —
> Reply to this email directly or view it on GitHub (https://github.com/sinatra/sinatra/pull/1025/files#r37137119).
>  
>  
>  
>  
>